v2/common/src/test/java/com/google/cloud/teleport/v2/utils/BigQueryIOUtilsTest.java

```diff
@@ -209,15 +210,22 @@ public void testWriteResultToBigQueryInsertErrorsWithStorageApiNoErrors() {
   }
 
   private WriteResult generateWriteResultsWithError(Pipeline pipeline, TableRow tableRow) {
-    WriteResult writeResult = pipeline.apply(
-            TestStream.create(TableRowJsonCoder.of())
-                .addElements(tableRow)
-                .advanceWatermarkToInfinity())
-        .apply(BigQueryIO.writeTableRows()
-            .withCreateDisposition(CreateDisposition.CREATE_NEVER)
-            .withExtendedErrorInfo()
-            .to(new TableReference().setProjectId(PROJECT).setDatasetId(DATASET).setTableId(TABLE))
-            .withTestServices(bigQueryServices));
+    WriteResult writeResult =
```

Comment:
nitpick/style: this can be simply returned instead of stored

Reply:
This file was reformatted when running spotless 
But, I don't know why you would store it in a variable. Changed it to return directly